apps/notifications/src/panel/boot/stylesheets/main.scss

```diff
@@ -538,14 +538,27 @@ $with-sidebar-min-page-width: 1114px;
 			text-align: left;
 			position: sticky;
 
-			.wpnc__settings {
+			.wpnc__settings,
+			.wpnc__keyboard-shortcuts-button {
 				display: flex;
 				flex-grow: 1;
 
 				&:focus svg {
 					outline: var(--color-primary-light) solid 2px;
 				}
 			}
+
+			.wpnc__keyboard-shortcuts-button {
+				flex-grow: 0;
+				padding-right: 8px;
+			}
+
+			.wpnc__keyboard-shortcuts-popover-anchor {
+				position: absolute;
+				right: 0;
+				top: 45px;
+				height: 18px;
+			}
```

Comment:
Here are the styles for the new anchor I added.

Reply:
Little worried about this as this position isn't always the same between calypso and standalone versions of the notifications panel. I'll test this out but I suspect we'll need to set it to the same position as the info icon

Reply:
> his position isn't always the same between calypso and standalone versions of the notifications panel.

Could you elaborate here?

>  I'll test this out but I suspect we'll need to set it to the same position as the info icon

That is definitely the goal!  I thought this was absolute positioning relative to the parent container of the panel which should make it consistent everywhere. I think if we wanted the exact same box as the icon I think we would add width: 18px and 8px of right margin or padding - but that seems unnecessary here.

Reply:
I didn't see any issue here when testing. I thought that this position could change slightly depending on where it rendered but haven't seen this in testing.